index.php

```diff
@@ -132,7 +132,7 @@ function get_q_value($mime, $accept)
 
 function base64_url_encode($string)
 {
-    $string = base64_encode($string);
+    $string = base64_encode($string ?? '');
```

Comment:
```suggestion
    $string = base64_encode(isset($string) ? $string : "");
```

Reply:
I never saw this comment, oops!  I have moved off my indieweb/selfauth project for a bit, but should hopefully be back to it soon and I can revisit this.